internal/initialize/initial.go

```diff
@@ -34,6 +35,9 @@ func init() {
 	fsSum.String("algorithm", algorithm, "hashing algorithm for hashing data")
 	fsSum.String("cm-name", configMapNameForHasher, "name of configMap for hasher")
```

Comment:
The configMap integrity-sum-config was removed from the deployment, correct? Then this flag is no longer needs.

Reply:
Good catch, will remove